project/plugins.sbt

```diff
@@ -1,4 +1,4 @@
 addSbtPlugin("org.scala-js"       % "sbt-scalajs"              % "1.18.2")
 addSbtPlugin("org.portable-scala" % "sbt-scalajs-crossproject" % "1.3.2")
 addSbtPlugin("com.github.sbt"     % "sbt-ci-release"           % "1.11.1")
-addSbtPlugin("org.scalameta"      % "sbt-scalafmt"             % "2.5.0")
```

Comment:
I was having trouble with scalafmt giving classpath errors but not easy to find out why, bumping to 2.5.1 sorted it but that is the latest we can use to keep Java8 compatibility